helm/templates/webhook/role.yaml

```
@@ -18,3 +18,10 @@ rules:
 - apiGroups: [""]
   resources: ["pods"]
   verbs: ["delete", "get", "list"]
+{{- if .Values.psp.enabled }}
+- apiGroups: ['policy']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
+  resourceNames:
+  - {{ .Values.psp.name }}
+{{- end}}
```

```suggestion
{{- if .Values.psp }}
{{- if .Values.psp.enabled }}
- apiGroups: ['policy']
  resources: ['podsecuritypolicies']
  verbs:     ['use']
  resourceNames:
  - {{ .Values.psp.name }}
{{- end}}
{{- end}}
```
same as in comment above